dashboard/test/helpers/levels_helper_test.rb

```diff
@@ -360,22 +366,23 @@ def request
     reset_view_options
   end
 
-  test 'use_google_blockly is false if Experiment is enabled but is_start_mode is true' do
-    @is_start_mode = true
+  test 'use_google_blockly is true if Experiment is enabled for google_blockly and @is_start_mode is false' do
+    @is_start_mode = false
     Experiment.stubs(:enabled?).returns(true)
-    @level = build :level
-    refute use_google_blockly
+    @level = build :level, type: 'GamelabJr'
+    assert use_google_blockly
     Experiment.unstub(:enabled?)
   end
 
-  test 'use_google_blockly is true if Experiment is enabled for google_blockly otherwise' do
+  test 'use_google_blockly is true if Experiment is enabled for google_blockly and it is not Sprite Lab start_mode' do
+    @is_start_mode = true
     Experiment.stubs(:enabled?).returns(true)
     @level = build :level
     assert use_google_blockly
     Experiment.unstub(:enabled?)
   end
 
-  test 'use_google_blockly is true if Experiment is enabled for google_blockly' do
+  test 'use_google_blockly is true if Experiment is enabled for google_blockly otherwise' do
```

Comment:
should `otherwise` be here?

Reply:
Nope! Resolved by merging staging into this branch.